### ISSUE TITLE
[Suggestion]: Ruined house over the water issue

### ISSUE THREAD

**Issue by Luke7381** [nonamecrackers2/crackers-wither-storm-mod#1570]
### Description of feature request

It can get annoying having to swim a couple of blocks or more when the ruin house spawns over water. Can there be like a bridge or just some exclusive structure for when it spawns in water? or just not make it spawn in water at all? My idea was the house to be some kind of underground base when over water but then any new player without knowledge of this mod wouldn't know what to do.

**Comment by Asteroid4** [comment]
i don't know if a bridge could work due to how structures are generated, maybe have it move to the nearest island? but then the player should probably be told where the structure is generated.

**Comment by myron-obsolete-account** [comment]
maybe I can make another structure for water biomes so it's not the plains one

**Comment by FalconinVoid** [comment]
> maybe I can make another structure for water biomes so it's not the plains one

Good Idea, it could just be a boat house for all ocean biomes. The boat could be like the ruined ones, but should swim.

**Comment by myron-obsolete-account** [comment]
okay so I made a structure
![image](https://github.com/nonamecrackers2/crackers-wither-storm-mod/assets/53058491/0e83c2c5-2a1d-4ca0-a915-4b470003c814)
I used a MC structure and built ontop of it:
+ big amulet out of wool, partially out of cobwebs, indicating it has worn down over decades or whatever
+ the summoning structure at the back
+ one golden apple stew in one of the loot chests
+ some uhh brewing stands
+ water

if cracker wants to use it (feel free to) I will send the nbt files here

**Comment by BluePineappleGuy** [comment]
nice, i like it. the amulet flag could use a little work, but its good

**Comment by FalconinVoid** [comment]
This is fr something we need! It would also fit on a beach! It could use some small detail changes here and there, but it is a beginning!

**Comment by NinjaSam5399** [comment]
This will come in the next event according to tcblizz.

**Comment by myron-obsolete-account** [comment]
> This will come in the next event according to tcblizz.

god damn it
